src/fpm_backend.f90

```diff
@@ -266,7 +266,12 @@ subroutine build_target(model,target,stat)
     select case(target%target_type)
 
     case (FPM_TARGET_OBJECT)
-        call run(model%fortran_compiler//" -c " // target%source%file_name // target%compile_flags &
+        if (model%implicit_none) then
+            implicit_flag = " -fimplicit-none"
+        else
+            implicit_flag = ""
+        end if
```

Comment:
This doesn't seem like the right place to add this option. At this point we are not aware which compiler we are using anymore and cannot be sure that we are actually passing the correct flag.

Reply:
Yes, see the first TODO item.